sdk/include/xdk/winnt_old.h

```diff
@@ -2560,9 +2560,13 @@ typedef struct _RTL_CRITICAL_SECTION_DEBUG {
   LIST_ENTRY ProcessLocksList;
   DWORD EntryCount;
   DWORD ContentionCount;
+#if defined(__WINESRC__) || defined(__HYBRIDSRC__)
+  DWORD_PTR Spare[2]; 
```

Comment:
This will break x64. It must be [8 / sizeof(DWORD_PTR)]. I am also generally unsure whether it's a good idea to use these 2 different interpretations, since it can easily lead to subtle bugs, when someone decides to use the fields correctly, but gets only bogus, and this is even different between x64 and x86. We should consider getting rid of the wine code that stores the string there and replacing it with an external function that does the coreect thing.